tests/dummy/app/controllers/application.js

```diff
@@ -88,6 +40,47 @@ export default Ember.Controller.extend({
     return `https://${window.location.hostname}/headset-troubleshooter`;
   }),
 
+  init () {
+    this._super(...arguments);
+    window.addEventListener('message', this.receiveMessage.bind(this), false);
+    this.fetchIceServers();
+  },
+
+  async fetchIceServers () {
+    const accessToken = this.get('application.authData.accessToken');
+    const streamingClient = await this.setupStreamingClient(accessToken);
+    const iceServers = await streamingClient.webrtcSessions.refreshIceServers();
+
+    console.log('received iceServers', JSON.stringify(iceServers));
+    this.set('iceServers', iceServers);
+    this.set('streamingClientConnected', true);
+
+    await streamingClient.disconnect();
```

Comment:
This is fine for now, but we'll have to rethink this in the future. Ideally, we'd disconnect after the tests have run but again we can fix that in the future. We want to add a test that makes a test call to voicemail and streaming client will obviously be needed for that.

Reply:
Oh, I completely forgot about that. It should be fairly easy to change in the future.